registration_form/registration_form.js

```diff
@@ -0,0 +1,48 @@
+function validUname(uname) {
+    let error = 0;
+    var length = uname.length;
+    var validChars = /\w{3,10}/;
+    document.getElementById('username_error').innerHTML = '';
+    if (length < 3 || length > 10 || !validChars.test(uname)) {
+        error++;
+        document.getElementById('username_error').innerHTML = "<em style='color:red'>Please enter valid username</em>";
```

Comment:
Nice. You may want to consider leaving `<em style='color:red'></em>` in the html itself and then populate only the innerText itself.